### PR TITLE
Add KaraFun elePHPant

### PR DESCRIPTION
**Pull request by cookieguru**

### CODE REVIEW

File: resources/data/elephpants.json [hunk 1]
```diff
@@ -519,6 +519,14 @@
...
             "year": 2021,
             "image": "https://i.imgur.com/NScLVf1.jpg"
+        },
+        {
+            "id": 66,
+            "name": "KaraFun",
+            "description": "KaraFun",
+            "sponsor": "KaraFun",
+            "year": 2022,
+            "image": null
```
Comment (derickr) [inline resources/data/elephpants.json:529]: Wouldn't it be nice to have an image for it?

Comment (cookieguru) [reply resources/data/elephpants.json:529]: It would, but I'm not aware of any images that have been granted usage rights for this project

Comment (ppelisset) [reply resources/data/elephpants.json:529]: Hello,
I work at KaraFun (the sponsor of this specimen). We have taken a photo of him and upload it at this URL: https://i.imgur.com/0TuGGoj.jpeg
We are ok for distribute photo under MIT Licence.
We would be happy to have our specimen present on elephpant.me

Comment (cookieguru) [reply resources/data/elephpants.json]: @ppelisset Thank you for the image!  I have updated the PR

**Review by ppelisset — APPROVED** [review]

**Review by jgrossi — APPROVED** [review]

**Pull request merged**